README.md

```diff
@@ -13,9 +13,139 @@ limitations under the License.
 
 [![License](https://img.shields.io/badge/License-Apache_2.0-lightgrey.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Triton Model Analyzer
-=====================
+# Triton Model Analyzer
 
-  **NOTE: You are currently on main-dev branch. This branch is tracking 
-  development for the new verssion of Model Analyzer and is not currently 
-  usable.**
+**NOTE: You are currently on main-dev branch. This branch is tracking 
+development for the new version of Model Analyzer and is not currently 
+usable.**
+
+Triton Model Analyzer is a CLI tool to help with better understanding of the
+compute and memory requirements of the Triton Inference Server models. These
+reports will help the user better understand the trade-offs in different
+configurations and choose a configuration that maximizes the performance of
+Triton Inference Server.
+
+The current release of the Triton Model Analyzer is 1.0.0 and corresponds to
+the 20.12 release of the tritonserver container on [NVIDIA GPU Cloud
+(NGC)](https://ngc.nvidia.com). The Docker tag that contains the Model Analyzer
+is 20.12-py3-sdk. The branch for this release is
+[r20.12](https://github.com/triton-inference-server/model_analyzer/tree/r20.12).
+
+## Requirements
+
+Triton Model Analyzer supports all the GPUs supported by the DCGM library.
+See [DCGM Supported GPUs](https://docs.nvidia.com/datacenter/dcgm/latest/dcgm-user-guide/getting-started.html#supported-platforms)
+for more information.
+
+## Quick Start
+
+The goal of this section is to analyze a simple libtorch model.
+
+1. Run Triton SDK Container:
+```
+docker run --net host -ti -v `pwd`/examples/quick-start:/workspace/examples nvcr.io/nvidia/tritonserver:20.12-py3-sdk
```

Comment:
This step is incorrect and not valid until after the 20.12 release. You will need to remove this option for now.

Reply:
Or replace this with a command to build the docker container locally